### PR TITLE
(Minor)Documentation update for Raisinbread

### PR DESCRIPTION
**Pull request by lbewsell**
## Brief summary of changes
Working through the raisin bread README I found a few items that need fixing. 

- [ Yes ] Have you updated related documentation?

#### Testing instructions (if applicable)

1. n/a

#### Link(s) to related issue(s)
n/a 
* Resolves #  (Reference the issue this fixes, if any.)

### CODE REVIEW

File: raisinbread/README.md [hunk 1]
```diff
@@ -68,8 +68,8 @@ directly on the mysql command line.
 
 ```bash
 cat SQL/0000-00-00-schema.sql \
-    SQL/0000-00-01-Modules.sql \
-    SQL/0000-00-02-Permission.sql \
+    SQL/0000-00-01-Permission.sql \
+    SQL/0000-00-02-Modules.sql \
```
Comment (ridz1208) [inline raisinbread/README.md:72]: @lbewsell as you can see in the SQL directory (https://github.com/aces/Loris/tree/main/SQL) the correct files are 
```
SQL/0000-00-01-Modules.sql \
SQL/0000-00-02-Permission.sql \
```

This is actually a change that took place in the latest version of LORIS which should be released soon. This documentation points to the latest updated information.

The changes happened here #7327

**Review by ridz1208 — CHANGES_REQUESTED** [review]
1 change needs to be undone. the other is valid and can be merged right after.

File: raisinbread/README.md (hunk 1 above)
Comment (lbewsell) [reply raisinbread/README.md:72]: Thanks for explaining that! I'm going to go ahead close this PR

**Comment by ridz1208** [comment]
@lbewsell I did incorporate your other change in https://github.com/aces/Loris/pull/7986. Thank you for that.

**Pull request closed without merging**